git-ssh.md

```diff
@@ -0,0 +1,75 @@
+# Installing SSH Key on Macbook for GitHub access
+
+I'm not going to walk through the reasoning behind using SSH keys as authorization or why GitHub is making this change. Let's just dive into the process and get the SSH keys created, installed, and referenced on GitHub.
+
+## Create the SSH Key
+1. Open terminal. **I recommend using the terminal application on your MacBook rather than the integrated terminal in VS Code.**
+1. Paste the following code into the terminal, substituting in your GitHub account email address for the `[your GitHub email]`:
+  ```
+  $ ssh-keygen -t -rsa -b 4096 -C "[your GitHub email]"
```

Comment:
I've found students are awful at replacing text in commands like this. I guarantee 1/2 the class will put their email in square brackets here 😜 

I might do:

-----
1. Paste the following code into the terminal, substituting "youremail@whatever.com" for your actual email address
  ```
  $ ssh-keygen -t -rsa -b 4096 -C "youremail@whatever.com"
  ```
  ----
  ....though I'm actually not sure if the email address matters for anything, so maybe not a big deal 🤷